### ISSUE TITLE
Not working with ruby 3.2

### ISSUE THREAD

**Issue by bcube90** [thecodecrate/city-state#54]
`if ! File.exists?` condition throws an error in the **Ruby version later or equal 3.2.0**

I was running it in `Rails 7.0.4` and the `Rails Console` suggests `Did you mean?  exist?`

**Comment by loureirorg** [comment]
@bcube90  Oh, thank you for letting me know that.

I apologize that I won't be able to check this issue until next weekend. I'll make sure to give it a thorough review as soon as I can.

**Comment by sabiou** [comment]
Yup it's not working. ran into same issue

**Comment by sabiou** [comment]
> `if ! File.exists?` condition throws an error in the **Ruby version later or equal 3.2.0**
> 
> I was running it in `Rails 7.0.4` and the `Rails Console` suggests `Did you mean? exist?`

Yup it's not working. ran into same issue

**Comment by kofiasare-dev** [comment]
Just run into this issue. When can  we expecting a fix?

**Comment by kstevens715** [comment]
I also just ran into this issue. I was able to work around it by pointing my Gemfile directly at GitHub because the issue was already fixed in d5da7f372e06065f5c25368836b9a6b820e92a63. It'd be great if an update to the gem could be released so it wasn't necessary to point right at `master` though.

```ruby
gem 'city-state', git: 'https://github.com/thecodecrate/city-state'
```

**Comment by loureirorg** [comment]
Sorry for the delay. It's fixed now on 1.1.0.
